Updated the Devices section (#2121)
Unity-Technologies/InputSystem: Packages/com.unity.inputsystem/Documentation~/simulate-touch-input.md

# Simulate touch input

> You can simulate touch input on other kinds of [pointer devices](pointers-introduction.md) such as [mouse](devices-mouse.md) and [pen](devices-pen.md) devices in the following ways:

Reviewer: I don't think "other kinds" makes sense here... perhaps it did in the previous page this part was taken from?

```suggestion
You can simulate touch input from [pointer devices](pointers-introduction.md) such as [mouse](devices-mouse.md) and [pen](devices-pen.md) devices in the following ways:
```
